docs/index.md

```diff
@@ -1628,6 +1628,7 @@ In addition to supporting the legacy [`mocha.opts`](#mochaopts) run-control form
 - **JavaScript**: Create a `.mocharc.js` in your project's root directory, and export an object (`module.exports = {/* ... */}`) containing your configuration.
 - **YAML**: Create a `.mocharc.yaml` (or `.mocharc.yml`) in your project's root directory.
 - **JSON**: Create a `.mocharc.json` in your project's root directory. Comments &mdash; while not valid JSON &mdash; are allowed in this file, and will be ignored by Mocha.
+- **[JSON5](https://json5.org/)**: Create a `.mocharc.jsonc` (or `.mocharc.json5`) in your project's root directory.
```

Comment:
Add the URL to the bottom of this file, not inline...

Reply:
@sstephant Please pull out the JSON5-related stuff.  Mocha shouldn't support it unless people explicitly ask for it.  Just "JSON with comments" and `.jsonc`, please.